models/servicenow.yml

```diff
@@ -1031,6 +1049,21 @@ models:
       
       - name: caller_id_value
         description: '{{ doc("caller_id_value")}}'
+
+      - name: caller_email
+        description: '{{ doc("email")}}'
+
+      - name: caller_manager_value
+        description: '{{ doc("manager_value")}}'
+
+      - name: caller_department_value
+        description: '{{ doc("department_value")}}'
+
+      - name: caller_name
+        description: '{{ doc("sys_user_name")}}'
+
+      - name: caller_roles
+        description: '{{ doc("roles")}}'
```

Comment:
Should we make it clear in the definitions that these are for the caller? I know the name of the field mentions caller, but it may be confusing with the definition of the field matching another field with a different name.

Reply:
Same comment for other types of fields that have this definition approach.

Reply:
I feel like it'll get repetitive, especially since the person that the field is regarding is stipulated in the field name itself. But I understand we can make it clearer. Perhaps instead of, for example, `department_value` = `Value representing the department associated with the system user.`, I can use the fact that the field name includes the user's role and update the description to 
`Value representing the department associated with the titular user.` That way the reader knows it's regarding the user in the field.

Let me know what you think, and will make the updates if that sounds good!